session/server.go

```diff
@@ -71,14 +71,17 @@ type Server struct {
 	activeSessions    map[sessionID]*mailboxSession
 	activeSessionsMtx sync.Mutex
 
+	sessionClosedSubs map[sessionID]chan struct{}
```

Comment:
Should we instead add a `quit` chan to the `mailboxSession` so we don't need another map?

Reply:
indeed. dont know why i didnt think of that. Thanks!